src/transaction/settings.ts

```diff
@@ -89,12 +90,16 @@ function createSettingsTransactionWithoutMemos(
   }
 
   if (settings.signers !== undefined) {
-    return {
+    const setSignerList = {
       TransactionType: 'SignerListSet',
       Account: account,
-      SignerQuorum: settings.signers.threshold,
-      SignerEntries: settings.signers.weights.map(formatSignerEntry)
+      SignerEntries: [],
+      SignerQuorum: settings.signers.threshold
+    };
+    if (settings.signers.weights !== undefined) {
+      setSignerList.SignerEntries = _.map(settings.signers.weights, formatSignerEntry);
```

Comment:
Is it necessary to use lodash here? We would actually like to remove the lodash dependency entirely. It adds a lot to the size of the library (especially the browser version) and there are always other ways to accomplish the same thing.